src/React.MSBuild/TransformJsx.cs

```diff
@@ -49,10 +49,11 @@ public override bool Execute()
 		{
 			MSBuildHost.EnsureInitialized();
 			var config = React.AssemblyRegistration.Container.Resolve<IReactSiteConfiguration>();
-			config
-				.SetReuseJavaScriptEngines(false)
-				.SetUseHarmony(UseHarmony)
-				.SetStripTypes(StripTypes);
+		    config
+		        .SetReuseJavaScriptEngines(false)
+		        .SetUseHarmony(UseHarmony)
+		        .SetStripTypes(StripTypes)
+		        .AddScript(string.Empty);
```

Comment:
Why is `string.Empty` necessary here? How does the code loading the scripts handle this?